community/modules/file-system/DDN-EXAScaler/main.tf

```diff
@@ -42,7 +42,7 @@ locals {
 }
 
 module "ddn_exascaler" {
-  source          = "github.com/DDNStorage/exascaler-cloud-terraform//gcp?ref=e33f439a"
+  source          = "github.com/DDNStorage/exascaler-cloud-terraform//gcp?ref=a3355d50deebe45c0556b45bd599059b7c06988d"
```

Comment:
nit: next time you can use `git rev-parse --short a3355d50deebe45c0556b45bd599059b7c06988d` to get something short but unique

Reply:
That's useful, thanks Tom!